src/Action.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component, PropTypes } from 'react';
+
+class Action extends Component {
```

Comment:
I'd be careful with the naming when using such words. An action usually refers to an actual action in software. So maybe `ActionHistoryItem` would be more descriptive.